eng/versioning/version_client.txt

```diff
@@ -89,6 +89,7 @@ com.azure:azure-core-experimental;1.0.0-beta.52;1.0.0-beta.53
 com.azure:azure-core-http-jdk-httpclient;1.0.0-beta.15;1.0.0-beta.16
 com.azure:azure-core-http-netty;1.15.3;1.16.0-beta.1
 com.azure:azure-core-http-okhttp;1.12.2;1.13.0-beta.1
+com.azure:azure-core-http-test-common;1.0.0-beta.1;1.0.0-beta.1
```

Comment:
I think this is unrelated? I don't find any reference to it and when I build the PR I don't see this show up.